### ISSUE TITLE
Liquibase 4.30 is causing re-run of all changeset

### ISSUE THREAD

**Issue by nitishxp** [liquibase/liquibase#6573]
### Search first

- [x] I searched and no similar issues were found

### Description

Hi Team,

I migrated from 4.29 to 4.30 and I am finding liquibase is re-running all the changeset again. I check on the databasechangelog and I am finding that new version(4.30) is using filename as id instead of raw. since my all existing changeset doesn't have author written on it

### Steps To Reproduce

If changeset doesn't contain any author upgrade from 4.29 to 4.30

### Expected/Desired Behavior

Liquibase shouldn't run the changeset

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by nitishxp** [comment]
Hi team,

Any update over this

**Comment by lbadam** [comment]
@nitishxp `id` and `author` are required fields for changesets ([docs](https://docs.liquibase.com/concepts/changelogs/changeset.html)). If those changed, they will re-run regardless of the Liquibase version.

**Comment by nitishxp** [comment]
@lbadam  I agree with your point but we have thousands of sql file where we don't have the id and author. And everything is fine upto 4.29 but with 4.30 id and author value is changed to file path causing all SQL to run again.
